common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1063,6 +1063,25 @@ class CourseModule(CourseFields, SequenceModule):  # pylint: disable=abstract-me
     """
 
 
+def validate_blackout_datetimes(blackout_dates):
+    """
+    Validates list of blackout dates.
+    """
+    date_proxy = Date()
+    if blackout_dates and type(blackout_dates[0]) not in (list, tuple):
+        blackout_dates = [blackout_dates]
+
+    ret = [
+        {"start": date_proxy.from_json(start), "end": date_proxy.from_json(end)}
+        for start, end
+        in [blackout_date for blackout_date in blackout_dates if blackout_date]
+    ]
+    for blackout in ret:
+        if not blackout["start"] or not blackout["end"]:
+            raise ValueError
+    return ret
```

Comment:
I think it might make sense to add the start < end validation here.

Reply:
Yeah. It was not there before. Now added check for that.